Sources/Dependencies/DependencyKey.swift

```diff
@@ -20,14 +22,6 @@ public protocol DependencyKey: TestDependencyKey {
   static var liveValue: Value { get }
 }
 
-extension DependencyKey {
-  /// A default implementation that provides the ``liveValue`` to Xcode previews.
-  public static var previewValue: Value { Self.liveValue }
-
-  /// A default implementation that provides the ``liveValue`` to tests.
-  public static var testValue: Value { Self.liveValue }
-}
-
```

Comment:
Moved this stuff down below since it got bigger and is just distracting if all you want to see are the protocol definitions.